packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -130,22 +130,24 @@ export const getServerSideProps = async function getServerSideProps(
   }
 
   const decidedAction = matchingRoute.action;
-
-  const { createContext } = await import("@calcom/trpc/server/createContext");
-  const ctx = await createContext(context);
-
-  const { default: trpcRouter } = await import("@calcom/app-store/routing-forms/trpc/_router");
-  const caller = trpcRouter.createCaller(ctx);
   const { v4: uuidv4 } = await import("uuid");
+
   let teamMembersMatchingAttributeLogic = null;
   let formResponseId = null;
   let attributeRoutingConfig = null;
   try {
-    const result = await caller.public.response({
-      formId: form.id,
-      formFillerId: uuidv4(),
-      response: response,
-      chosenRouteId: matchingRoute.id,
+    const { responseHandler } = await import("../../trpc/response.handler");
+    const result2 = await responseHandler({
+      ctx: {
+        prisma,
+        serializableForm,
```

Comment:
This file was also initialising serializedForm, which would then be done again in the responseHandler - as this is the primary expensive DB operation I've moved this to the responseHandler context as an optional context param - so there's less overhead when using the routing-forms router as an embed